packages/xod-arduino/platform/configuration.tpl.cpp

```diff
@@ -15,6 +15,12 @@
 // Uncomment to trace the program runtime in the Serial Monitor
 //#define XOD_DEBUG_ENABLE_TRACE
 
+{{#if XOD_USERNAME}}
+#define XOD_USERNAME "{{XOD_USERNAME}}"
+{{else if XOD_USERNAME_NEEDED}}
+//#define XOD_USERNAME "your_username"
+static_assert(false, "The program uses the `=XOD_USERNAME` literal. Uncomment the definition of the `XOD_USERNAME` variable and change the value to your username and remove this line.");
```

Comment:
```suggestion
static_assert(false, "Program uses `=XOD_USERNAME` literal. Uncomment line above, put your username there, and remove this line.");
```

Suggest shorter version: it’s a compiler message which should be as short as possible